platforms/shared/tests/test-client/main.cpp

```diff
@@ -23,6 +23,8 @@ using namespace cv;
 
 namespace prc = prism::connect;
 
+typedef boost::chrono::system_clock::time_point time_point;
```

Comment:
My intention was to provide shortcut. Any suggestion on improving this? May be use "using".
Also, we agreed to create a new task in Jira, if comment is about code already merged.
I will write separate mail to discuss a code review, because it seems the focus is wrong during the code review. And this is not the first time I have such thoughts.